libcontainer/configs/cgroup_unix.go

```diff
@@ -16,6 +16,9 @@ type Cgroup struct {
 	// name of parent cgroup or slice
 	Parent string `json:"parent"`
 
+	// CgroupsPath specifies the path to cgroups that are created and/or joined by the container.
+	CgroupsPath string `json:"cgroupsPath"`
```

Comment:
hmm i think we can just start removing them here ? I think it's also clearer when we integrate this with docker too since there will be complication errors :p